develop/README.md

```diff
@@ -0,0 +1,32 @@
+# Docker
+
+## Installing ONLYOFFICE Docs
+
+## How to use - Linux
+
+**Note**: You need the latest Docker version installed.
+
+You might need to pull **onlyoffice/documentserver** image:
+
+```bash
+sudo docker pull onlyoffice/documentserver
```

Comment:
Do not prefix `docker` command with `sudo`
[This](https://docs.docker.com/engine/install/linux-postinstall/#manage-docker-as-a-non-root-user) instruction show how to use docker without `sudo`